app/views/uploaded_files/index.html.haml

```diff
@@ -0,0 +1,23 @@
+%h1 Listing uploaded_files
+
+%table
+  %thead
+    %tr
+      %th Title
+      %th Description
+      %th
+      %th
+      %th
+
+  %tbody
+    - @uploaded_files.each do |uploaded_file|
+      %tr
+        %td= uploaded_file.title
+        %td= uploaded_file.description
+        %td= link_to 'Show', uploaded_file
+        %td= link_to 'Edit', edit_uploaded_file_path(uploaded_file)
+        %td= link_to 'Destroy', uploaded_file, :method => :delete, :data => { :confirm => 'Are you sure?' }
```

Comment:
It used to be that you had to use the old hashrocket with that (passing the delete method and data in).   That may have changed; maybe it can be written in the newer style.  (someone needs to check and can change this)